### ISSUE TITLE
Notebook doesn't accept pyzmq 14.5.0

### ISSUE THREAD

**Issue by tommis** [ipython/ipython#7922]
When I run ipython notebook I get this.

    (ipython)[tms@themisto Code]$ ipython notebook
    Traceback (most recent call last):
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/utils/zmqrelated.py", line 10, in check_for_zmq
        import zmq
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/__init__.py", line 49, in <module>
        from zmq import backend
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/backend/__init__.py", line 41, in <module>
        reraise(*exc_info)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/utils/sixcerpt.py", line 34, in reraise
        raise value
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/backend/__init__.py", line 29, in <module>
        _ns = select_backend(first)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/backend/select.py", line 27, in select_backend
        mod = __import__(name, fromlist=public_api)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/zmq/backend/cython/__init__.py", line 6, in <module>
        from . import (constants, error, message, context,
    ImportError: libzmq.so.4: cannot open shared object file: No such file or directory
    
    During handling of the above exception, another exception occurred:
    
    Traceback (most recent call last):
      File "/home/tms/.virtualenvs/ipython/bin/ipython", line 11, in <module>
        sys.exit(start_ipython())
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/__init__.py", line 120, in start_ipython
        return launch_new_instance(argv=argv, **kwargs)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 573, in launch_instance
        app.initialize(argv)
      File "<string>", line 2, in initialize
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 75, in catch_config_error
        return method(app, *args, **kwargs)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/terminal/ipapp.py", line 321, in initialize
        super(TerminalIPythonApp, self).initialize(argv)
      File "<string>", line 2, in initialize
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 75, in catch_config_error
        return method(app, *args, **kwargs)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/core/application.py", line 369, in initialize
        self.parse_command_line(argv)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/terminal/ipapp.py", line 316, in parse_command_line
        return super(TerminalIPythonApp, self).parse_command_line(argv)
      File "<string>", line 2, in parse_command_line
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 75, in catch_config_error
        return method(app, *args, **kwargs)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 471, in parse_command_line
        return self.initialize_subcommand(subc, subargv)
      File "<string>", line 2, in initialize_subcommand
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 75, in catch_config_error
        return method(app, *args, **kwargs)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/config/application.py", line 402, in initialize_subcommand
        subapp = import_item(subapp)
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/utils/importstring.py", line 42, in import_item
        module = __import__(package, fromlist=[obj])
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/html/notebookapp.py", line 29, in <module>
        check_for_zmq('13', 'IPython.html')
      File "/home/tms/.virtualenvs/ipython/lib/python3.4/site-packages/IPython/utils/zmqrelated.py", line 12, in check_for_zmq
        raise ImportError("%s requires pyzmq >= %s"%(required_by, minimum_version))
    ImportError: IPython.html requires pyzmq >= 13

    (ipython)[tms@themisto Code]$ pip freeze
    bash-kernel==0.3
    ipython==3.0.0
    Jinja2==2.7.3
    MarkupSafe==0.23
    pexpect==3.3
    pyzmq==14.5.0

**Comment by Carreau** [comment]
maybe dumb question but do you have libzmq installed though distrib packet manager ?

**Comment by minrk** [comment]
How did you install pyzmq, and on what platform? What happens when you do `python -c 'import zmq'`?

**Comment by tommis** [comment]
Turns out it wasn't a dumb question. I tried getting ipython notebook running the same way I do without a virtualenv. I try to run notebook continously and install the package it fails importing until it runs. I got confused because it failed on a pyzmq import but I had installed it on virtualenv.
To fix it I also installed ipython from package repo.

There still might be bug because maybe it uses the system version when it should use virtualenv one.
